requirements-dev.txt

```diff
@@ -1,3 +1,7 @@
+scipy
+numpy
+sympy
+qiskit>=2.0.0
```

Comment:
`scipy`, `numpy` and `qiskit` should all already be direct dependencies of Aer - does everything work if these are removed from the list?

Reply:
They are in the setup.py.
However, there might be some value in having a more strict set of testing dependencies compared to the required dependencies.  This can often be useful in tracking down build problems across time.

Reply:
It seems removing `numpy` and `scipy` from the requirements doesn't have an adverse effect, so I'll keep them out for now.